### PR TITLE
Update bundlephobia-search extension

### PR DESCRIPTION
**Pull request by rushelex**
## Description

I decided not to wait for someone to make [this issue](https://github.com/raycast/extensions/issues/14502) and made it myself.

### TL;DR
1. Added information about Tree Shaking
2. Updated the `@raycast/api` library + all dependent libraries
3. Updated screenshots (created via [Raycast Screenshots Maker by gebeto](https://github.com/gebeto/raycast-screenshots-maker))
4. Replaced the icon with a more beautiful one (IMHO) and consistent with other Raycast icons. The icon was created in the [Icon Maker by Raycast](https://ray.so/icon) service

## Screencast

<details>
<summary>Updated screenshots</summary>

screenshot-1
screenshot-2
</details>

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by dgrzedzielski — APPROVED** [review]
LGTM 🎉 

Thanks for the contribution 🙌

**Comment by rushelex** [comment]
@dgrzedzielski I think you should merge this PR because you are a contributor and I don't have permission to do that

**Comment by dgrzedzielski** [comment]
> @dgrzedzielski I think you should merge this PR because you are a contributor and I don't have permission to do that

Hey I also have no permissions, we need to wait for raycast team to merge

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥
